core/src/main/java/io/questdb/cairo/ColumnPurgeJob.java

```diff
@@ -253,6 +268,14 @@ private void putTasksFromTableToQueue() {
                     if (taskRun != null) {
                         retryQueue.add(taskRun);
                     }
+
+                    if (count > columnPurgeQueueLimit) {
+                        // There can be too many tasks in the table so that this code can cause OOM
+                        // and QuestDB will not be able to start at all. Stop here and let the user know
+                        LOG.critical().$("too many Column Purge Tasks in the table some task will be ignored, " +
```

Comment:
I am not sure alerting someone at 2am makes any sense when we could have automatically removed unused items on startup, synchronously.

Reply:
done